setup.py

```diff
@@ -23,7 +24,7 @@
 
 setup(
     name='easypost',
-    version='5.1.0',
+    version='5.1.1',
```

Comment:
It seems like it's too easy for this to get out of sync with version.py. Maybe this should be automatically populated?

Reply:
Good suggestions. Perhaps a unit test or something.

An earlier version was trying to to some introspection, but it messed with packaging/setuptools.